src/stories/HsitorySelect/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import {Button, ButtonProps, HistorySelectionDialogTitle, SelectionModal} from '../../components';
+
+export interface HistorySelectionProps extends ButtonProps{}

```

Comment:
no need to have it here, therefore ButtonProps and, HistorySelectionDialogTitle can be removed from imports.